desktop-app/src/renderer/components/Previewer/Device/index.tsx

```diff
@@ -450,33 +515,58 @@ const Device = ({ isPrimary, device, setIndividualDevice }: Props) => {
         device={device}
         setScreenshotInProgress={setScreenshotInProgess}
         openDevTools={openDevTools}
+        toggleRuler={toggleRuler}
         onRotate={onRotateHandler}
         onIndividualLayoutHandler={onIndividualLayoutHandler}
         isIndividualLayout={isIndividualLayout}
       />
       <div
-        style={{ height: scaledHeight, width: scaledWidth }}
+        style={{
+          height: scaledHeight,
+          width: scaledWidth + 30,
+        }}
         className="relative origin-top-left bg-white"
       >
-        <webview
-          id={device.name}
-          src={address}
-          style={{
-            height,
-            width,
-            display: 'inline-flex',
-            transform: `scale(${zoomfactor})`,
-          }}
-          ref={ref}
-          className="origin-top-left"
-          /* eslint-disable-next-line react/no-unknown-property */
-          preload={`file://${window.responsively.webviewPreloadPath}`}
-          data-scale-factor={zoomfactor}
-          /* eslint-disable-next-line react/no-unknown-property */
-          allowpopups={isPrimary ? ('true' as any) : undefined}
-          /* eslint-disable-next-line react/no-unknown-property */
-          useragent={device.userAgent}
+        <GuideGrid
+          scaledHeight={scaledHeight}
+          scaledWidth={scaledWidth}
+          height={height}
+          width={width}
+          coordinates={coordinates}
+          zoomFactor={zoomfactor}
+          night={darkMode}
+          enabled={rulerEnabled(`${width}x${height}`)}
+          defaultGuides={window.electron.store
+            .get('userPreferences.guides')
+            .flatMap((x: any) => x)
+            .filter((x: DefaultGuide) => {
+              return x.resolution === `${width}x${height}`;
+            })}
         />
+        <div style={{}} className="bg-slate-200 dark:bg-slate-800">
+          <webview
```

Comment:
It looks like the default `white` background style is not working any more? 

<img width="526" alt="Screenshot 2023-11-25 at 7 42 46 PM" src="https://github.com/responsively-org/responsively-app/assets/1283424/5d3f57ae-b15f-4643-a8c4-0f018c38ab0e">